### DISCUSSION TITLE
Instance Oracle?

### DISCUSSION THREAD

**Discussion by Andrefiel** [IceWhaleTech/CasaOS#1045] [General]
Please, did anyone manage to use CASAOS together with an Oracle ARM instance?

**Comment by yesbhautik** [comment]
I am, i am using

**Reply by Andrefiel** [reply]
When you did the installation by command, did you have to release which port or did you do some extra procedure to access the settings interface?

**Reply by yesbhautik** [reply]
> When you did the installation by command, did you have to release which port or did you do some extra procedure to access the settings interface?

what issue are actually facing?

**Comment by yesbhautik** [comment]
Yes bro, DM me on WhatsApp will explain you
https://wa.me/916353586391
